aspnet/mvc/views/forms/sample/final/ViewModels/AddressViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace FormsTH.ViewModels
+{
+    public class AddressViewModel
+    {
+        public string AddressLine1 { get; set; }
+    }
+
+    public class RegisterViewModel2
+    {
+        public string Email { get; set; }
+
+        public string Password { get; set; }
```

Comment:
@dougbu this is the start of the tutorial and I'm only using one TH, the Form (so I get the token). The HTML at this stage is:
``Password: <input type="password" id="Password" name="Password" />``
I'm using the formula "start out simple then build up". In the next section I introduce data annotations and apply 
```
 [DataType(DataType.Password)]
        public string Password { get; set; } ```